frontend/assets/l10n/app_en.json

```diff
@@ -96,7 +96,9 @@
     "cameraAccessRequiredSettings": "To scan a QR code, the app needs access to the camera. You can allow the app to access the camera in the settings.",
     "cardAlreadyActivated": "The scanned QR code has already been activated.",
     "cardExpired": "Your card has expired. You can apply for renewal under \"More actions\"",
+    "cardExpiredBeforeActivation": "This card has expired.",
```

Comment:
I suggest: "Your card has already expired and can no longer be activated. Please submit a new application."